README.md

```diff
@@ -1,125 +1,190 @@
+it's commonplace for Cadence NFTs to store metadata onchain while EVM NFTs often store an onchain pointer to metadata
+stored offchain. In order for Cadence NFTs to be properly represented in EVM platforms, the metadata must be bridged in
+a format expected by those platforms and be done in a manner that also preserves the atomicity of bridge requests. The
+path forward on this was decided to be a commitment of serialized Cadence NFT metadata into formats popular in the EVM
+ecosystem.
+
+For assets that do not implement `EVMBridgedMetadata`, the bridge will attempt to serialize the metadata of the asset as
+a JSON data URL string. This is done via the [`SerializeMetadata`
+contract](./cadence/contracts/utils/SerializeMetadata.cdc) which serializes metadata values into a JSON blob compatible
+with the OpenSea metadata standard. The serialized metadata is then committed as the ERC721 `tokenURI` upon bridging
+Cadence-native NFTs to EVM. Since Cadence NFTs can easily update onchain metadata either by field or by the ownership of
+sub-NFTs, this serialization pattern enables token URI updates on subsequent bridge requests.
+
+### Opting Out
+
+It's also recognized that the logic of some use cases may actually be compromised by the act of bridging, particularly
+in such a unique runtime environment. These would be cases that do not maintain ownership assumptions implicit to
+ecosystem standards. For instance, an ERC721 implementation may reclaim a user's assets after a month of inactivity time
+period. In such a case, bridging that ERC721 to Cadence would decouple the representation of ownership of the bridged
+NFT from the actual ownership in the definining ERC721 contract after the token had been reclaimed - there would be no
+NFT in escrow for the bridge to transfer on fulfillment of the NFT back to EVM. In such cases, projects may choose to
+opt-out of bridging, but **importantly must do so before the asset has been onboarded to the bridge**.
+
+For Solidity contracts, opting out is as simple as extending the [`BridgePermissions.sol` abstract
+contract](./solidity/src/BridgePermissions.sol) which defaults `allowsBridging()` to false. The bridge explicitly checks
+for the implementation of `IBridgePermissions` and the value of `allowsBridging()` to validate that the contract has not
+opted out of bridging.
+
+Similarly, Cadence contracts can implement the [`IBridgePermissions.cdc` contract
+interface](./cadence/contracts/bridge/interfaces/IBridgePermissions.cdc). This contract has a single method
+`allowsBridging()` with a default implementation returning `false`. Again, the bridge explicitly checks for the
+implementation of `IBridgePermissions` and the value of `allowsBridging()` to validate that the contract has not opted
+out of bridging. Should you later choose to enable bridging, you can simply override the default implementation and
+return true.
```

Comment:
It would be good to get this added to the V2 standards so we can show that every NFT contract needs to be cognizant about if they want to allow bridging or not. Can you make a PR to the v2 standard branch?